### PR TITLE
Throughtspot release

### PR DESCRIPTION
**Pull request by MonikaFeigler**

### CODE REVIEW

**Comment by MonikaFeigler** [comment]
templates aren't tested, because cli doesn't work.

File: .keboola/repository.json [hunk 1]
```diff
@@ -34,6 +34,66 @@
...
+            "keboola.ex-facebook-ads",
+            "keboola.ex-google-ads",
+            "keboola.ex-sklik",
+            "keboola.wr-snowflake-blob-storage",
+            "keboola.wr-thoughtspot"
+          ],
+          "path": "v0"
+        }
+      ]
     }
```
Comment (JakubMatejka) [inline .keboola/repository.json:97]: Tahle šablona tu nemá folder.

**Comment by MonikaFeigler** [comment]
I've deleted templates which are blocked by oauth and prepared a release of templates which are available without oauth

**Comment by JakubMatejka** [comment]
@MonikaFeigler Michal told me that you should find someone who will try the templates and check end to end that they work. (See https://keboolaglobal.slack.com/archives/CHPE0FPU5/p1654499247276689) Unfortunately, we don't have capacity to do it ourselves.

**Pull request closed without merging**